src/styles/getTooltipStyles.js

```diff
@@ -100,18 +100,16 @@ export default function getTooltipStyles(
     // We get wrapper width based on the tooltip's inner text width so the wrapper is just big enough to fit text and prevent white space.
     // If the text width is less than the maximum available width, add horizontal padding.
     // Note: tooltipContentWidth ignores the fractions (OffsetWidth) so add 1px to fit the text properly.
-    const wrapperWidth = tooltipContentWidth && tooltipContentWidth < maxWidth
-        ? tooltipContentWidth + (spacing.ph2.paddingHorizontal * 2) + 1
-        : maxWidth;
+    const wrapperWidth = tooltipContentWidth && tooltipContentWidth + (spacing.ph2.paddingHorizontal * 2) + 1;
```

Comment:
I don't see why we are making a change here either

Reply:
Based on the other explanation, I think I see the change here, the first render will always set the width to max width and this is unwanted. Please mark this as resolved if it's the case.

Reply:
yeah that was the case

Reply:
Hi, @s77rt @hannojg It seems this change caused [this issue](https://github.com/Expensify/App/issues/15949).  Do you guys agree to remove this change to resolve this regression as I mentioned in [the proposal](https://github.com/Expensify/App/issues/15949#issuecomment-1479806759)
cc @eVoloshchak

Reply:
No. The change is intended to have tooltips behave correctly on custom rendered content (non-texts). I will look into the issue as well.